Tests/secrets_white_list.json

```diff
@@ -2672,7 +2672,10 @@
     "vulnInfoList",
     "NAS-Identifier=de-wilm-251littl-idf3b-s2",
     "NAS-Port-Id=GigabitEthernet2/0/05",
-    "--memory=1g##--memory-swap=-1##--pids-limit=256##--ulimit=nofile=1024:8192"
+    "--memory=1g##--memory-swap=-1##--pids-limit=256##--ulimit=nofile=1024:8192",
+    "oproxy-dev.shared_views.top_packs",
+    "Pack.EXCLUDE_DIRECTORIES",
+    "Author_image.png"
```

Comment:
Why are these added?

Reply:
Cause `demisto-sdk secrets` failed, but now it failed on other stuff